google-cloud-spanner/src/main/java/com/google/cloud/spanner/MetricRegistryConstants.java

```diff
@@ -36,9 +36,14 @@
   private static final LabelValue UNSET_LABEL = LabelValue.create(null);
 
   static final LabelValue NUM_IN_USE_SESSIONS = LabelValue.create("num_in_use_sessions");
+
+  @Deprecated
```

Comment:
nit: could we add javadocs on the deprecation notice?

Reply:
Done.